src/EntityHelper.ts

```diff
@@ -1,23 +1,79 @@
-import { User, Token } from "../generated/schema"
+import { BigDecimal, BigInt } from "@graphprotocol/graph-ts";
+import { User, Token, DailySwapVolume, FeePerDay, AstDailySnapshot } from "../generated/schema";
+import { _ERC20 } from "../generated/SwapContract/_ERC20";
+import { Transfer } from "../generated/AirswapToken/AirswapToken";
 
 export function getUser(userAddress: string): User {
-  let user = User.load(userAddress)
-  // handle new creation of User if they don't exist
-  if (!user) {
-    user = new User(userAddress)
-    user.authorizeAddress = new Array<string>()
-    user.save()
+  let user = User.load(userAddress);
+
+  if (user == null) {
+    user = new User(userAddress);
+    user.tokenBalance = BigInt.fromI32(0);
+    // user.tokenBalanceUSD = BigDecimal.fromString('0');
+    user.totalTransfers = BigInt.fromI32(1);
+    user.authorizeAddress = new Array<string>();
+      user.save();
   }
-  return user as User
+
+  return user as User;
 }
 
 export function getToken(tokenAddress: string): Token {
-  let token = Token.load(tokenAddress)
+  let token = Token.load(tokenAddress);
   // handle new creation of Token if it doesn't exist
   if (!token) {
-    token = new Token(tokenAddress)
-    token.isBlacklisted = false
-    token.save()
+    token = new Token(tokenAddress);
+    token.isBlacklisted = false;
```

Comment:
How does `isBlacklisted` factor in?

Reply:
Just shows if the token is blacklisted, can be removed if not needed.